packages/material-ui/src/NativeSelect/NativeSelectInput.js

```diff
@@ -77,24 +77,19 @@ const SelectRoot = experimentalStyled(
     '&:focus': {
       borderRadius: theme.shape.borderRadius, // Reset the reset for Chrome style
     },
-    '&&': {
+    '&&&': {
```

Comment:
For bumping the specificity we need to use now specificity of 3, as naturally the style overrides for the slot would override these definitions